project/src/components/place-card/place-card.tsx

```diff
@@ -2,41 +2,49 @@ import Badge from '../badge/badge';
 import Bookmark from '../bookmark/bookmark';
 import {Link} from 'react-router-dom';
 import {AppRoute} from '../../const/routing';
-import { MULTIPLER } from '../../const/general';
+import {MAX_RATING} from '../../const/general';
 import {Offer} from '../../types/offers';
 
 type PlaceCardProps = {
   className: string;
   offer: Offer;
+  onMouseOver: (cardId: number) => void;
 }
 
-function PlaceCard({className, offer}: PlaceCardProps): JSX.Element {
+function PlaceCard(props: PlaceCardProps): JSX.Element {
+  const {className, offer, onMouseOver} = props;
+  const {isPremium, id, previewImage, title, price, rating, description, type} = offer;
+
+  const cardItemMouseHoverHandler = () => {
+    onMouseOver(id);
+  };
+
   return (
-    <article className={`${className} place-card`}>
-      {offer.isPremium ? <Badge className="place-card__mark" /> : ''}
+    <article className={`${className} place-card`} onMouseOver={cardItemMouseHoverHandler}>
```

Comment:
Здесь можно не выносить отдельно 
```suggestion
    <article className={`${className} place-card`} onMouseOver={() => onCardHover(id)}>
```

Reply:
ok